src/sentry/seer/code_review/webhooks/issue_comment.py

```diff
@@ -0,0 +1,92 @@
+"""
+Handler for GitHub issue_comment webhook events.
+"""
+
+from __future__ import annotations
+
+import logging
+from collections.abc import Mapping
+from typing import Any
+
+from sentry import features
+from sentry.integrations.github.client import GitHubReaction
+from sentry.integrations.services.integration import RpcIntegration
+from sentry.models.organization import Organization
+from sentry.models.repository import Repository
+from sentry.utils import metrics
+
+from ..permissions import has_code_review_enabled
+
+logger = logging.getLogger(__name__)
+
+SENTRY_REVIEW_COMMAND = "@sentry review"
+
+
+def is_pr_review_command(comment_body: str | None) -> bool:
+    if comment_body is None:
+        return False
+    return SENTRY_REVIEW_COMMAND in comment_body.lower()
+
+
+def _add_eyes_reaction_to_comment(
+    integration: RpcIntegration | None,
+    organization: Organization,
+    repo: Repository,
+    comment_id: str,
+) -> None:
+    if integration is None:
+        metrics.incr("seer.code_review.webhook.issue_comment.reaction_missing_integration")
```

Comment:
I tend to use a prefix so I can easily create error widgets in Datadog.
https://github.com/getsentry/sentry/blob/40560ae6d40a4289ee2bf2c34755ce4db50944f4/src/sentry/seer/code_review/webhooks/check_run.py#L44-L45